package.json

```diff
@@ -126,7 +126,7 @@
     "webpack-bundle-analyzer": "^3.7.0",
     "webpack-cli": "^3.3.10",
     "webpack-merge": "^4.2.2",
-    "yargs": "^15.1.0",
+    "yargs": "^15.3.1",
```

Comment:
This is necessary because we use a feature introduced in `yargs@15.2.0` so could get a version that satisfied our semver range but actually blew up during the tests.